index.js

```diff
@@ -1,61 +1,186 @@
+'use strict';
+
 const core = require('@actions/core');
 const github = require('@actions/github');
 const http = require('@actions/http-client');
 
-async function main() {
-    let sha = github.context.sha;
+function GitlabClient (inputConfig, githubConfig) {
```

Comment:
I assume that this works but it looks weird to me to create a function and use it as a class. Can/should this be a class instead?

Reply:
TBH last time I was using JS that was the only syntax :) It's called object constructor function and it was a standard way of creating objects. But now I see JS has a class keyword, so I'll rewrite it. Thanks :+1:

Reply:
I know it's not a crucial change, but it's so easy to do, that there is no reason not to change it and learn something :)